extensions/gdx-tools/src/com/badlogic/gdx/tools/texturepacker/MaxRectsPacker.java

```diff
@@ -129,7 +130,7 @@ private Page packPage (Array<Rect> inputRects) {
 					throw new RuntimeException("Image does not fit within max page width " + settings.maxWidth + paddingMessage + ": "
 						+ rect.name + " " + width + "x" + height);
 				}
-				if (height > maxHeight && (!settings.rotation || width > maxHeight)) {
```

Comment:
@lucas-viva Is there a reason to remove these checks?

Reply:
@NathanSweet This was explained in https://github.com/libgdx/libgdx/pull/7453#discussion_r1740999019